elements/pf-select/pf-select.ts

```diff
@@ -0,0 +1,517 @@
+import { LitElement, html, type PropertyValues } from 'lit';
+import { customElement } from 'lit/decorators/custom-element.js';
+import { property } from 'lit/decorators/property.js';
+import { query } from 'lit/decorators/query.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { classMap } from 'lit/directives/class-map.js';
+import { type ListboxValue } from '@patternfly/pfe-core/controllers/listbox-controller.js';
+import { ToggleController } from '@patternfly/pfe-core/controllers/toggle-controller.js';
+import type { PfSelectOption } from './pf-select-option.js';
+import { PfChipGroup } from '@patternfly/elements/pf-chip/pf-chip-group.js';
+import { PfSelectList } from './pf-select-list.js';
+
+import styles from './pf-select.css';
+
+export type PfSelectItemsDisplay = '' | 'badge' | 'chips';
```

Comment:
```suggestion
export type SelectItemsKind = 'badge' | 'chips';
```
if there's a third type, give it a name that's not `''`